.changeset/breezy-toes-help.md

```diff
@@ -0,0 +1,5 @@
+---
+'eslint-plugin-qwik': major
```

Comment:
major is no good, as this package is linked to all the rest (qwik, qwik-city) which will make them go to V2 :)

So this needs to be either minor, or if it has breaking changes - should go into the V2 branch

Thanks!